style.css

```diff
@@ -111,3 +111,8 @@ p {
 .custom-card .contact img {
   width: 40px;
 }
+
+.marius-quote{
```

Comment:
Please remove those CSS changes, as they are not necessary. Wrap this part in `<em>` and `<strong>` tags and you will have the same result 😄